evalytics/server/communications_channels.py

```diff
@@ -0,0 +1,44 @@
+from googleapiclient.discovery import build
```

Comment:
too many `@classmethod` for my body.  I'd use regular `self` methods instead to be honest. 

```python
gmail_channel.send('peter.parker@gmail.com', data)
```

sounds good IMO
```